receiver/hostmetricsreceiver/internal/scraper/processscraper/metadata.yaml

```diff
@@ -46,6 +46,10 @@ resource_attributes:
     description: The username of the user that owns the process.
     enabled: true
     type: string
+  process.cgroup:
+    description: cgroup associated with the process.
```

Comment:
We should mention that this is only available on Linux systems, right?

Reply:
In semantic conventions we are trying to specify system specific metrics under a specific namespace. For example in this scraper, `process.handles` will soon become `process.windows.handles` since it's Windows exclusive. So I imagine we will want to name this `system.linux.cgroup`. It would probably be good to do that here, because that's probably what we'd call it anyway for semconv.

Reply:
:shrug: cgroups indeed only exist under Linux. I can add that to the description.